src/components/Tooltip/__stories__/tooltip.stories.mdx

```diff
@@ -0,0 +1,205 @@
+import Tooltip from "../Tooltip";
+import TooltipReference from "./TooltipReference";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import { Hide, Subitems, Bolt } from "../../Icon/Icons"
+import Icon from "../../Icon/Icon.jsx"
+import Button from "../../Button/Button.jsx";
+import { modifiers } from "./helper.js";
+import { Link } from "../../../storybook/components";
+import { CHIP, LABEL, TIPSEEN } from "../../../storybook/components/related-components/component-description-map";
+import "./tooltip.stories.scss";
+
+<Meta
+  title="Components/Tooltip"
+  component={ Tooltip }
+/>
+
+<!--- Component template -->
+
+export const tooltipTemplate = (args) => {
+  return (
+    <div className="monday-storybook-tooltip_overview">
+      <Tooltip
+        shouldShowOnMount={args.shouldShowOnMount}
+        content={args.content}
+        modifiers={args.modifiers}
+      >
+        <div />
+      </Tooltip>
+    </div>
+  )
+}
+
+<!--- Component documentation -->
+
+# Tooltip
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+Tooltips show contextual help or information about specific components when a user hovers on them.
+<Canvas>
+  <Story name="Overview"
+         args={ {shouldShowOnMount: true, modifiers: modifiers, content: "I’m a tooltip"} }>
```

Comment:
after set the modifiers in the template function you can remove the modifiers arguments